### PR TITLE
Revue50 rotating log files

### PR DESCRIPTION
**Pull request by mikeumus**
From @Revue50's #284

### CODE REVIEW

**Comment by mikeumus** [comment]
Hey @skreutzberger, what do you think? 🤔

**Comment by skreutzberger** [comment]
Looks good! You could turn `noOfLogFiles` and `maxLogFilesize` into public variables so that they can be adjusted from the outside. Also please add tests covering the logic.

**Comment by skreutzberger** [comment]
@mikeumus did you see my recent comment on the PR?

**Comment by skreutzberger** [comment]
@mikeumus! Please turn `noOfLogFiles` and `maxLogFilesize` into public variables so that they can be adjusted from the outside. Also please add tests covering the logic. Once that is done I will merge the pull request.

File: Sources/FileDestination.swift [hunk 1]
```diff
@@ -89,10 +91,54 @@ public open class FileDestination: BaseDestination {
...
+           for index in stride(from: lastIndex, to: firstIndex, by: -1) {
+               let oldFile = String.init(format: "%@.%d", filePath, index)
+               if FileManager.default.fileExists(atPath: oldFile) == true {
+                   if index == lastIndex {
+                       // Delete the last file
+                       try FileManager.default.removeItem(atPath: oldFile)
+                   } else {
+                       // Move the current file to next index
+                       let newFile = String.init(format: "%@.%d", filePath, index+1)
+                       try FileManager.default.moveItem(atPath: oldFile, toPath: newFile )
```
Comment (Ewg777) [inline Sources/FileDestination.swift:131]:
```
toPath: newFile )
    ||
    \/
toPath: newFile)
```

**Pull request merged**